packages/cfd/src/Containers/cfd-password-modal/icon-type.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { CFD_PLATFORMS, routes } from '@deriv/shared';
+import TradingPlatformIcon from '../../Assets/svgs/trading-platform';
+
+type TIconTypeProps = {
+    platform: string;
+    type?: string;
+    show_eu_related_content: boolean;
+};
+
+export const IconType = React.memo(({ platform, type, show_eu_related_content }: TIconTypeProps) => {
+    const traders_hub = window.location.pathname === routes.traders_hub;
+    if (platform === CFD_PLATFORMS.DXTRADE) {
+        return <Icon icon='IcRebrandingDxtradeDashboard' size={128} />;
+    } else if (platform === CFD_PLATFORMS.DERIVEZ) {
+        return <Icon icon='IcBrandDerivEz' size={128} />;
+    } else if (traders_hub) {
+        if (platform === CFD_PLATFORMS.CTRADER) {
+            return <TradingPlatformIcon icon='CTrader' size={128} />;
+        }
+        switch (type) {
+            case 'synthetic':
```

Comment:
We need to replace these values with enums. So if we need to change anything we do it from one file.
```suggestion
            case MARKET_TYPE.SYNTHETIC:
```
packages/cfd/src/Helpers/cfd-config.ts